tools/clang/test/CodeGenHLSL/quick-test/const_init.hlsl

```diff
@@ -0,0 +1,11 @@
+// RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
+
+// Make sure default val for cbuffer element is ignored.
+// CHECK: fadd
```

Comment:
Would be good to add a check for a warning that the initializer is unused.

Reply:
Test don't check warnings if compile successes. Will support it in another pull request.